Export/Exporter.php

```diff
@@ -44,16 +44,17 @@ public function __construct(SerializerInterface $serializer)
     /**
      * Returns a serialized/exported version of the specified object.
      *
-     * @param object      $object     The object to be exported
-     * @param string|null $objectName The name of the object used in the export (eg. the root node in XML)
-     * @param string      $format     The format an object is serialized to
+     * @param object      $object         The object to be exported
+     * @param string|null $objectName     The name of the object used in the export (eg. the root node in XML)
+     * @param string      $format         The format an object is serialized to
+     * @param string|null $targetEncoding The target encoding for the export (XML)
      *
      * @return string
      *
      * @throws InvalidArgumentException
      * @throws ReflectionException
      */
-    public function exportObject(/*object */$object, $objectName = null, $format = 'xml')
+    public function exportObject(/*object */$object, $objectName = null, $format = 'xml', $targetEncoding = null)
```

Comment:
I would suggest to make the default targetEncoding UTF-8.
```suggestion
    public function exportObject(/*object */$object, $objectName = null, $format = 'xml', $targetEncoding = 'UTF-8')
```

Reply:
Done. I wanted to maintain the behavior of existing implementations, but UTF-8 is a safe default.